immuni_exposure_ingestion/monitoring/celery.py

```diff
@@ -0,0 +1,38 @@
+from prometheus_client import Counter, Gauge
+
+from immuni_common.monitoring.core import NAMESPACE, Subsystem
+
+KEYS_PROCESSED = Counter(
+    namespace=NAMESPACE,
+    subsystem=Subsystem.API.value,
+    name="keys_processed",
+    documentation="Total number of keys processed.",
+)
+
+BATCHES_CREATED = Counter(
+    namespace=NAMESPACE,
+    subsystem=Subsystem.API.value,
+    name="batches_created",
+    documentation="Total number of Batches created.",
```

Comment:
`... created batches.`?

Reply:
If you want to use the capital Batches, then I think it's more appropriate to be consistent with the Mongo model, BatchFiles.